mesonbuild/compilers/c.py

```diff
@@ -89,13 +89,12 @@ def get_options(self):
         opts = CCompiler.get_options(self)
         c_stds = ['c89', 'c99', 'c11']
         g_stds = ['gnu89', 'gnu99', 'gnu11']
-        if self.compiler_type is CompilerType.CLANG_OSX:
```

Comment:
This if statement may have been in error. I don't see anything in Clang docs distinguishing MacOS Clang from other OS for the `-std=` flag.